packages/@react-spectrum/avatar/docs/Avatar.mdx

```diff
@@ -0,0 +1,65 @@
+<!-- Copyright 2021 Adobe. All rights reserved.
```

Comment:
just a heads up but all docs will need to be pulled out into another PR eventually, up to you when you do it